docs/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix.md

```diff
@@ -283,19 +283,21 @@ Run this test to make sure it passes. In Visual Studio, open the **Test Explorer
 
 ## Create tests for valid declarations
 
-As a general rule, analyzers should exit as quickly as possible, doing minimal work. Visual Studio calls registered analyzers as the user edits code. Responsiveness is a key requirement. There are several test cases for code that should not raise your diagnostic. Your analyzer already handles one of those tests, the case where a variable is assigned after being initialized. Add the following test method to represent that case:
+As a general rule, analyzers should exit as quickly as possible, doing minimal work. Visual Studio calls registered analyzers as the user edits code. Responsiveness is a key requirement. There are several test cases for code that should not raise your diagnostic. Your analyzer already handles several of those tests. Add the following test methods to represent those cases:
 
 [!code-csharp[variable assigned](snippets/how-to-write-csharp-analyzer-code-fix/MakeConst/MakeConst.Test/MakeConstUnitTests.cs#VariableAssigned "a variable that is assigned after being initialized won't raise the diagnostic")]
 
-This test passes as well. Next, add test methods for conditions you haven't handled yet:
+[!code-csharp[already const declaration](snippets/how-to-write-csharp-analyzer-code-fix/MakeConst/MakeConst.Test/MakeConstUnitTests.cs#AlreadyConst "a declaration that is already const should not raise the diagnostic")]
```

Comment:
Can you change all the snippet references in this file to use the `:::` directive instead? That's the current standard. Also, use the `Id=` element with a snippet comment tag rather than the `lines=` element.